server/meshmodel/policies/network.json

```diff
@@ -0,0 +1,21 @@
+{
+  "apiVersion": "core.meshery.io/v1alpha1",
+  "kind": "Network",
+  "metadata": {
+    "description": "This policy validates the network configuration in a Kubernetes environment, ensuring that service and container ports, protocols, and mappings are correctly defined and consistent. It helps maintain secure and reliable network communication between services and pods, preventing misconfigurations and potential vulnerabilities",
+    "name": "Kubernetes Network Policy Validator",
+    "version": "v1.0.0"
+  },
+  "model": {
+    "name": "core",
```

Comment:
The below policy should come under kubernetes, the core version of this policy is under darft.

Reply:
Roger that. I was following how we have defined our [meshmodel relationships](https://github.com/meshery/meshery/tree/master/server/meshmodel/relationships).

Should I change the model to `Kubernetes` in this PR or you are mentioning this as a future change?

Reply:
Yeah, change the model to `kubernetes`. The `core` version would be there in some forth-coming PRs.